src/pheval/runners/runner.py

```diff
@@ -3,6 +3,14 @@
 from dataclasses import dataclass
 from pathlib import Path
 
+from pheval.config_parser import parse_input_dir_config
+from pheval.utils.constants import (
+    PHEVAL_GENE_RESULTS_DIR,
+    PHEVAL_VARIANT_RESULTS_DIR,
+    TOOL_INPUT_COMMANDS_DIR,
+    TOOL_RESULTS_DIR,
+)
```

Comment:
I guess Jules will tell you to add these as member vars into the class, but good luck.

Reply:
For real! These aren't used anywhere else and only have context inside that class, so why externalise them?